src/cdk/v2/destinations/zoho/transformRecordV2.js

```diff
@@ -209,11 +209,18 @@ const handleDeletion = async (
   input,
   identifiers,
   Config,
```

Comment:
config and destConfig what is the difference

Reply:
do this for remaining functions also

Reply:
Config is from `destination.Config` and destConfig is `connection.config`.